Automattic/wp-calypso
# Fix the social icons in the post list preview

Completes 1203820933396971-as-1204692223892775/f

## Proposed Changes

* Fix the color for Instagram icon
* Fix the border-radius for Facebook icon

## Testing Instructions

- Goto `/posts/:site` for a site with Instagram and a Facebook account connected
- Click on 3 dots for a post and click on Share
- In the Social Media accounts list, confirm that Instagram account has a proper icon visible
- Confirm that Facebook icon is rounded

| Before | After |
|--------|--------|
| <img width="303" alt="Screenshot 2023-06-05 at 2 50 36 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/9bd443bb-2b1f-40f3-8e85-eb4b3128238b"> | <img width="306" alt="Screenshot 2023-06-05 at 3 28 35 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/0568893e-b863-4b96-8a75-34bab8c112fd"> | 

## Pre-merge Checklist
